src/Tribe/RSVP.php

```diff
@@ -1842,6 +1842,9 @@ public function get_ticket( $event_id, $ticket_id ) {
 		$return->report_link      = '';
 		$return->show_description = $return->show_description();
 
+		// All tickets are 'none' by default, ET+ will filter to change.
+		$return->iac = 'none';
```

Comment:
This really should be a constant from the Abstract, so we avoid having the string being passed around.

Reply:
I set it on the ticket object, can remove this

Reply:
Removed